boot.php

```diff
@@ -14,7 +14,7 @@
 }
 
 // bessere SEO urls für den image-manager
-if (!rex::isBackend()) {
```

Comment:
in dieser art würde man den check machen wenn yrewrite optional für dein addon ist.

wenn yrewrite integraler bestandteil ist und ohne yrewrite dieses addon nicht funktioniert solltest du das requirement in der package.yml deklarieren (siehe andere FOR addons)